app/controllers/api/base_controller/renderer.rb

```diff
@@ -193,7 +193,7 @@ def collection_filterer(res, type, klass, is_subcollection = false)
 
       def filter_results(miq_expression, res, options)
         if miq_expression.present? && options.key?(:limit) && options.key?(:offset)
-          subquery_res = Rbac.filtered(res, options.except(:offset, :limit))
+          subquery_res = Rbac.filtered(res, options.except(:offset, :limit, :extra_cols))
           [Rbac.filtered(res, options), subquery_res.count]
```

Comment:
would this work?

```suggestion
          [Rbac.filtered(res, options), subquery_res.count(:all)]
```

Reply:
@kbrock Not unless this would work:

```ruby
irb> [1,2,3].count(:all)
#=> 0
```

Reply:
Resolving since this will not be something we are going to use.